### DISCUSSION TITLE
GoPro Hero 12 GoPro Lab Add an Overlay to Video is not working

### DISCUSSION THREAD

**Discussion by vans0862** [gopro/labs#596] [Q&A]
https://gopro.github.io/labs/control/overlays/ 

I have GP8, GP11 and GP12. 
GP8, GP11 are work but GP12 is not working. 
Does anyone know if it works? Or is it not working yet?

**Comment by dnewman-gpsw** [comment]
It works in 4K30 16x9 8-bit or lower resolution. I expect you have 10-bit enabled.

**Reply by vans0862** [reply]
![IMG_8982](https://github.com/gopro/labs/assets/83860284/f302c49e-181a-4121-9289-6e12c0709001)
![IMG_8983](https://github.com/gopro/labs/assets/83860284/7006cf47-1471-4b0c-91e8-1d19c5972d11)
![IMG_8984](https://github.com/gopro/labs/assets/83860284/84c925e2-603c-4a15-9a59-39ab3da39298)
![IMG_8985](https://github.com/gopro/labs/assets/83860284/a8d0eb0a-6242-40e1-9cc0-237c13be8f93)
![IMG_8986](https://github.com/gopro/labs/assets/83860284/e9f7121a-3a2b-4f83-94c9-19914462a680)

**Reply by vans0862** [reply]
I know It works in 4K30 16x9 8-bit or lower resolution but my GP12 is not work. Is my GP12 problem? should I return it?

**Reply by dnewman-gpsw** [reply]
No to the return. Labs is never a reason to return a camera.  Labs is buggy by design (almost) so you probably just found a corner case where it doesn't work.  In testing 1080p60 SV here, it does work on the HERO12 (all of them), however there could be other factors that are interfering in your setup.  Share a short clip that has overlays enabled, but doesn't work (share via dropbox or similar.) I can check the metadata.

**Comment by vans0862** [comment]
I will keep my gopro11 and 8. 
I sold gopro12.
